core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -302,8 +302,8 @@ package object config {
     .createWithDefaultString("1g")
 
   private[spark] val EXECUTOR_MEMORY_OVERHEAD = ConfigBuilder("spark.executor.memoryOverhead")
-    .doc("The amount of non-heap memory to be allocated per executor in cluster mode, " +
-      "in MiB unless otherwise specified.")
+    .doc("The amount of non-heap memory to be allocated per executor, in MiB unless otherwise" +
```

Comment:
This change means that the non-cluster mode refers to this value?

Reply:
yes, the 'client' deploy mode respects this too